jax/_src/compiler.py

```diff
@@ -91,6 +91,22 @@ def use_detailed_logging(module: ir.Module) -> bool:
   return _walk_operations(module.operation, bound) < 0
 
 
+def log_persistent_cache_hit(module_name: str) -> None:
+  hit_log_priority = (logging.WARNING if config.log_compiles.value
+                      else logging.DEBUG)
+  logger.log(hit_log_priority, "Persistent compilation cache hit for '%s'",
+             module_name)
+
+
+def log_persistent_cache_miss(module_name: str) -> None:
+  miss_log_priority = (logging.WARNING
+                        if config.explain_cache_misses.value
+                        and compilation_cache.is_persistent_cache_enabled()
+                        else logging.DEBUG)
+  logger.log(miss_log_priority, "PERSISTENT COMPILATION CACHE MISS for '%s'",
```

Comment:
nit: maybe comment this is all caps to match trace cache miss logging, future readers may wonder :)